checkstyle-tester/pom.xml

```diff
@@ -1,71 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-	<modelVersion>4.0.0</modelVersion>
-
-	<groupId>com.github.sevntu.checkstyle</groupId>
-	<artifactId>sample</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
-	<packaging>jar</packaging>
-
-	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<checkstyle-plugin.version>3.1.1</checkstyle-plugin.version>
-		<checkstyle.version>0.0-SNAPSHOT</checkstyle.version>
-		<sevntu-checkstyle.version>1.44.1</sevntu-checkstyle.version>
```

Comment:
Sevntu support is completely broken until #340 / #604 . We only supported single mode runs anyways, never supported differences since the sevntu version was never changed.